### ISSUE TITLE
The given configuration works with versions 3.0.1 and 3.0.2, writing json logs to the specified path. However, it does not work with version 3.0.3

### ISSUE THREAD

**Issue by Midhun458** [apache/logging-log4net#211]
Conf ;
```
<threshold value="WARN"/>
<file type="log4net.Util.PatternString" value="path" />
<appendToFile value="true" />
<layout type="log4net.Layout.SerializedLayout, log4net.Ext.Json">
	<decorator type="log4net.Layout.Decorators.StandardTypesDecorator, log4net.Ext.Json" />
	<default />
	<!--explicit default members-->
	<remove value="ndc" />
	<remove value="message" />
	<!--remove the default preformatted message member-->
	<member value="clientid:clientid" />
	<member value="message:messageobject" />
	<!--add raw message-->
</layout>
```

Windows .net framework 4.7.2  log4net.Ext.Json version 3.0.1. it suppose to write json to the defined path.

**Comment by gdziadkiewicz** [comment]
I will take a look

**Comment by gdziadkiewicz** [comment]
@Midhun458 I reproduced it and confirmed that it started with log4net 3.0.3. Work on the fix is in progress.

**Comment by gdziadkiewicz** [comment]
Hi @Midhun458  , I implemented a fix in log4net.Ext.Json 3.0.3-develop1560902832 . It resolves the problem on my side; can you test it on yours?

**Comment by gdziadkiewicz** [comment]
@Midhun458 did you manage to try it out?

**Comment by Midhun458** [comment]
![image](https://github.com/user-attachments/assets/3e853197-7f91-4e77-ba09-2961b85bda9a)
This does write log without any issue.

**Comment by gdziadkiewicz** [comment]
@Midhun458 I released full-fledged 3.0.3 version of log4net.Ext.Json
@FreeAndNil we can close the issue

**Comment by FreeAndNil** [comment]
@gdziadkiewicz thanks a lot.
